packages/common/src/utils/categoryUtils.ts

```diff
@@ -0,0 +1,33 @@
+import { eq } from 'drizzle-orm/sql';
+import { isEmpty } from 'lodash-es';
+
+import { Dependencies } from '../config/config.js';
+import { NewCategory, dictionaryCategories } from '../models/dictionary_categories.js';
+import categoryRepository from '../repository/categoryRepository.js';
+
+const utils = (dependencies: Dependencies) => {
+	return {
+		createCategoryIfDoesNotExist: async (categoryName: string) => {
+			const { logger } = dependencies;
+			try {
+				const categoryRepo = categoryRepository(dependencies);
+				const foundCategory = await categoryRepo.select({}, eq(dictionaryCategories.name, categoryName));
+				if (!isEmpty(foundCategory)) {
+					logger.info(`Category ${categoryName} already exists. Not doing any action`);
+					return foundCategory[0];
```

Comment:
Unlike the other time I called out array access, we know this is not undefined because of the isEmpty check. This one should be good.